dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -44,6 +44,30 @@ Sounds.prototype.play = function (soundId, options) {
   sound.play(options);
 };
 
+Sounds.prototype.playURL = function (url, options) {
+  // Play a sound given a URL, register it using the URL as id and infer
+  // the file type from the extension at the end of the URL
+  // (NOTE: not ideal because preload happens inside first play)
+  var sound = this.soundsById[url];
+  if (sound) {
+    sound.play(options);
+  } else {
+    var config = {id: url};
+    var ext = url.substr(url.lastIndexOf('.') + 1);
+    config[ext] = url;
+    // Force HTML5 audio if the caller requests it (cross-domain origin issues)
+    config.forceHTML5 = options && options.forceHTML5;
+    // Force HTML5 audio on mobile if the caller requests it
+    config.allowHTML5Mobile = options && options.allowHTML5Mobile;
+    // since preload may be async, we set playAfterRegister in the config so we
+    // play the sound once it is loaded
+    // Also stick our play() options inside the config as firstPlayOptions
+    config.playAfterRegister = true;
+    config.firstPlayOptions = options;
```

Comment:
Hm, feels like these two parameters should live together somehow, i.e., would be odd if firstPlayOptions existed but playAfterRegister didn't.

Simple Option A: parallel naming, `playAfterLoad` and `playAfterLoadOptions`

Option B: collapse these two into a parameter `playWithOptionsOnLoad`, the existence of which implies playback. Don't quite love that since you could ostensibly just want to use the default parameters, which feels a bit off.